INFO  | File run_exp_CCN_Project.py | Line 15    | Writing log file to ./exp\GTS_test_1223_002238\log_exp_1010.txt
INFO  | File run_exp_CCN_Project.py | Line 16    | Exp instance id = GTS_test_1223_002238
ERROR | File run_exp_CCN_Project.py | Line 25    | Traceback (most recent call last):
  File "C:\Users\KimJuHyeon\GSL\run_exp_CCN_Project.py", line 20, in main
    runner.train()
  File "C:\Users\KimJuHyeon\GSL\runner\CCN_Project_runner.py", line 168, in train
    _, pred = torch.max(angle.data, 1)
IndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)

